istioctl/pkg/writer/envoy/configdump/listener.go

```diff
@@ -291,7 +312,7 @@ func (c *ConfigWriter) PrintListenerSummary(filter ListenerFilter) error {
 		return iType < jType
 	})
 
-	printStr := "ADDRESS\tPORT"
+	printStr := "ADDRESS\tPORT\tADDITIONAL ADDRESSES"
```

Comment:
I don't think we should add this part. Its adding a lot of bloat to the output without providing much value.

if we really need to put all of them, I would just use the same column with CSV:

```
ADDRESS
0.0.0.0,::
```

Reply:
> I don't think we should add this part. Its adding a lot of bloat to the output without providing much value.
> 
> if we really need to put all of them, I would just use the same column with CSV:
> 
> ```
> ADDRESS
> 0.0.0.0,::
> ```

Hi @howardjohn & @jacob-delgado, if the address contains `port`, it may look like this:

```
ADDRESS
0.0.0.0:80,[::]:80
```